services/wallet/transfer/sequential_fetch_strategy.go

```diff
@@ -0,0 +1,90 @@
+package transfer
+
+import (
+	"context"
+	"math/big"
+	"sync"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/event"
+	"github.com/status-im/status-go/rpc/chain"
+	"github.com/status-im/status-go/services/wallet/async"
+)
+
+type SequentialFetchStrategy struct {
+	db                 *Database
+	blockDAO           *BlockDAO
+	feed               *event.Feed
+	mu                 sync.Mutex
+	group              *async.Group
+	transactionManager *TransactionManager
+	chainClients       map[uint64]*chain.ClientWithFallback
+	accounts           []common.Address
+}
+
+func (r *SequentialFetchStrategy) newCommand(chainClient *chain.ClientWithFallback,
+	// accounts []common.Address) *loadAllTransfersCommand {
+	accounts []common.Address) async.Commander {
+
+	signer := types.NewLondonSigner(chainClient.ToBigInt())
+	// ctl := &loadAllTransfersCommand{
+	ctl := &controlCommand{ // TODO Will be replaced by loadAllTranfersCommand in upcoming commit
+		db:          r.db,
+		chainClient: chainClient,
+		accounts:    accounts,
+		blockDAO:    r.blockDAO,
+		eth: &ETHDownloader{
+			chainClient: chainClient,
+			accounts:    accounts,
+			signer:      signer,
+			db:          r.db,
+		},
+		erc20:              NewERC20TransfersDownloader(chainClient, accounts, signer),
+		feed:               r.feed,
+		errorsCount:        0,
+		transactionManager: r.transactionManager,
+	}
+	return ctl
+}
+
+func (s *SequentialFetchStrategy) start() error {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+
+	if s.group != nil {
+		return errAlreadyRunning
+	}
+	s.group = async.NewGroup(context.Background())
+
+	// Run the command on interval (5 seconds)
+	for _, chainClient := range s.chainClients {
```

Comment:
does it mean we load all transfer for all chain with interval between chain at 5s?